src/auro-popover.js

```diff
@@ -115,7 +162,7 @@ class AuroPopover extends LitElement {
     return html`
       <div id="popover" class="popover util_insetLg">
         <div id="arrow" class="arrow" data-popper-arrow></div>
-        <slot role="tooltip"></slot>
+        <slot name="tooltip"></slot>
```

Comment:
Why was this changed? 

The purpose of `role` is for accessibility. This also made the use of the component require to have to use a named slot whereas the generic slot would have populated this element prior.

Reply:
Reverted.